systemProperties.adoc

```diff
@@ -0,0 +1,26 @@
+== System Properties
+Below, a number of relevant System Properties are listed.
+
+
+NOTE: With `getBoolean`, returns if a System Property with the provided {@code systemPropertyKey} either exists, is set to "yes" or is set to "true", see
```

Comment:
the @code doesn't work in adoc - see https://github.com/OpenHFT/Chronicle-Network/blob/Issue_192/systemProperties.adoc